File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        neg_counter = 0
        for i in nums:
            if i < 0:
                neg_counter += 1
            elif i == 0:
                return 0
        if neg_counter % 2 != 0:
            return -1
        
        return 1
        

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.go
func arraySign(nums []int) int {
    neg_counter := 0
    for i:=0;i<len(nums);i++{
        if nums[i] == 0{
            return 0
        } else if nums[i] < 0{
            neg_counter++
        }
    }
    if neg_counter % 2 != 0{
        return -1
    }
    
    return 1
}